helm/README.md

```diff
@@ -0,0 +1,64 @@
+# source{d} blog deployment
+
+This is the [Helm](https://helm.sh/) chart to deploy [source{d} blog](https://blog.sourced.tech) in kubernetes.
+
+# Introduction
+
+The blog website is generated using [Hugo](http://gohugo.io/)
+and served with [Caddy](https://caddyserver.com/) web server.
+A docker container is built by our self hosted [Drone](https://drone2.srcd) server,
+and deployed using this [helm plugin](https://github.com/ipedrazas/drone-helm) in our GKE kubernetes clusters.
+
```

Comment:
Although interesting, Drone is not related to this chart. I would place this information in a different place.